app/components/VerifyTab.tsx

```diff
@@ -16,6 +16,33 @@ export function VerifyTab() {
                 {
                   path: ['$.given_name'],
                 },
+                {
+                  path: ['$.family_name'],
+                },
+                {
+                  path: ['$.age_equal_or_over.21'],
+                  filter: {
+                    type: 'boolean',
+                    const: true,
```

Comment:
Not sure whether this needs a `const: true`

Reply:
depends, do we want to check whether the value is true, or do we just want the value?